Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Description: ensure BLAS include dir is added to kernel BUILD_INTERFACE
-------------------------------------------------------------------------------------------
diff -ruN pastix-v6.2.2_p2/kernels/CMakeLists.txt pastix-v6.2.2_p3.1/kernels/CMakeLists.txt
--- pastix-v6.2.2_p2/kernels/CMakeLists.txt	2022-09-09 17:31:31.000000000 +0100
+++ pastix-v6.2.2_p3.1/kernels/CMakeLists.txt	2022-09-26 15:24:08.000000000 +0100
@@ -151,6 +151,7 @@
   $<BUILD_INTERFACE:${PASTIX_BINARY_DIR}/include>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+  $<BUILD_INTERFACE:${CBLAS_DIR}/include>
   $<INSTALL_INTERFACE:include> )
 
 target_link_libraries( pastix_kernels PUBLIC spm            )
